.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
+  - name: GTFS.org
+    url: https://gtfs.org/
+    about: The official reference for GTFS documentation. 
+  - name: GTFS Changes Google Group
```

Comment:
I'd call this GTFS Changes Mailing List (this is what is used in the [Spec Amendment Process](https://gtfs.org/schedule/process/)) and more self-explanatory.

Reply:
I agree. Let's keep it close to what is used in Amendment Process.